eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -740,7 +740,7 @@ protected function getRealAliasId($locationId)
         $data = $this->gateway->loadAutogeneratedEntry('eznode:' . $locationId);
 
         // Root entries (URL wise) can return 0 as the returned value is used as parent (parent is 0 for root entries)
-        if (empty($data) || $data['id'] != 0 && $data['parent'] == 0 && strlen($data['text']) == 0) {
+        if (empty($data) || ($data['id'] != 0 && $data['parent'] == 0 && strlen($data['text']) == 0)) {
```

Comment:
This looks like a bug fix, not just improvement. Seems sane, but I hope you've checked that.

Reply:
Not really, rather code style - PHP interprets the code fragment this way.

Reply:
There's a flame war going on for ages about using brackets for expressions that are evaluated due to precedence. From my side +0.5 on this change ;)

Reply:
I'll certainly take logical grouping over relying on arbitrary precedence rules, any day. +1. 🔥